docs/components/camera/image-file.md

```diff
@@ -9,8 +9,7 @@ tags: ["camera", "components"]
 # SMEs: Bijan, vision team
 ---
 
-An `image_file` camera gets color and depth images frames from a file path.
-If you provide a depth image, as well as the intrinsic parameters of the camera that took that depth image, then the camera will also provide a point cloud.
```

Comment:
This second line is still true. There are two ways of producing point clouds now:

1. provide a path to a pcd file (new way)
2. provide a path to a depth image, as well as the intrinsic parameters of the camera that took the depth image.

If you do both, the pcd file takes precedence

Reply:
Ok, thank you for explaining this! Will update and request for re-review once addressed

Reply:
Think this is addressed now but I moved the usage tip below, lmk if any issues